vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentalGroupsLayout.java

```diff
@@ -84,12 +89,30 @@ public interface ExperimentalGroupCommandListener {
 
   }
 
+  private static class BaseExperimentalGroupCard extends VerticalLayout {
 
-  private static class ExperimentalGroupCard extends VerticalLayout {
+    public final static String CARD_WIDTH = "200px";
+    public final static String CARD_HEIGHT = "200px";
 
-    private final static String CARD_WIDTH = "200px";
-    private final static String CARD_HEIGHT = "200px";
+    public BaseExperimentalGroupCard() {
+      setWidth(CARD_WIDTH);
+      setHeight(CARD_HEIGHT);
+      getStyle().set("margin", "5px 5px");
+      setCardLayoutStyle();
+    }
+
+    private void setCardLayoutStyle() {
+      addClassNames(
+          "rounded-m",
+          "box-border",
+          "rounded-m",
+          "shadow-xs",
+          "p-m"
+      );
+    }
+  }
```

Comment:
Why was this moved into a seperate Class? The Width and Height is hard set so it can't be reused in other classes. An Idea would be to allow the user to set the width and height so it can be used in other components?

Reply:
the new class is used because the card with groups and the card with the "add group" button were using a lot of the same code. I don't foresee a third use case for these specific cards but with other sizes at the moment.